### ISSUE TITLE
Can not create schema reference when deploying

### ISSUE THREAD

**Issue by zaniar** [apioo/fusio#103]
I have my schema in `json` file. I want to reference to other schema. So I added the following lines:
```json
"$ref": "schema:///SchemaName"
```
When I execute `deploy` command, it gave me message:

```
An exception occurred during deploy. No changes are applied to the database.

Message: Scheme schema is not supported
Trace: #0 vendor\fusio\impl\src\Service\System\Deploy\Transformer\Schema.php(134): Fusio\Impl\Service\System\Deploy\Transformer\Schema->traverseSchema(Object(stdClass), '...')
#1 vendor\fusio\impl\src\Service\System\Deploy\Transformer\Schema.php(112): Fusio\Impl\Service\System\Deploy\Transformer\Schema->traverseSchema(Object(stdClass), '...')
#2 vendor\fusio\impl\src\Service\System\Deploy\Transformer\Schema.php(80): Fusio\Impl\Service\System\Deploy\Transformer\Schema->resolveRefs('...')
#3 vendor\fusio\impl\src\Service\System\Deploy\Transformer\Schema.php(69): Fusio\Impl\Service\System\Deploy\Transformer\Schema->resolveSchema('!include resour...', '...')
#4 vendor\fusio\impl\src\Service\System\Deploy\Transformer\Schema.php(59): Fusio\Impl\Service\System\Deploy\Transformer\Schema->transformSchema('Unified', '!include resour...', '...')
#5 vendor\fusio\impl\src\Service\System\Deploy.php(95): Fusio\Impl\Service\System\Deploy\Transformer\Schema->transform(Array, Object(stdClass), '...')
#6 vendor\fusio\impl\src\Console\System\DeployCommand.php(109): Fusio\Impl\Service\System\Deploy->deploy(Array, '...')
#7 vendor\symfony\console\Command\Command.php(264): Fusio\Impl\Console\System\DeployCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 vendor\symfony\console\Application.php(874): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 vendor\symfony\console\Application.php(228): Symfony\Component\Console\Application->doRunCommand(Object(Fusio\Impl\Console\System\DeployCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 vendor\symfony\console\Application.php(130): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 bin\fusio(47): Symfony\Component\Console\Application->run()
#12 {main}
```

I tried the same thing but using the backend app and it worked as expected. No issue at all.

**Comment by k42b3** [comment]
Hi, currently when you deploy a schema only the `file://` scheme is allowed to reference another schema. But actually I think it would be good to support also the `schema://` scheme. Maybe we add also support for `http://` which gets then resolved on deploy.

Initially we have made this limitation because if you use `schema://` you must have a specific order of your schemas, so that you import schemas with dependencies at the end. But I think it is better to allow this.

**Comment by k42b3** [comment]
This is now implemented in the v1.0.0-RC5 release
